playbooks/allocate_pubcloud.yml

```diff
@@ -130,7 +130,7 @@
             image: "{{ _image | default(image) }}"
             key_name: "{{ keyname }}"
             security_groups: []
-            userdata: "{{ lookup('file', user_data_path) }}"
+            userdata: "{{ lookup('file', playbook_dir ~ '/../scripts/user_data_pubcloud.sh') }}"
```

Comment:
It might have been nice to define a var instead so they're more likely to be in sync.